build.sbt

```diff
@@ -22,6 +22,13 @@ scalacOptions ++= {
     Seq("-target:jvm-1.7")
 }
 
+lazy val sangriaOsgiSettings = osgiSettings ++ Seq(
+  OsgiKeys.exportPackage := Seq("sangria.*;version=${Bundle-Version}"),
+  OsgiKeys.privatePackage := Seq()
+)
+
+lazy val sangriaProject = project.in(file(".")).enablePlugins(SbtOsgi).settings(sangriaOsgiSettings:_*)
```

Comment:
To follow the existing style this can be simplified to:

```sbt
enablePlugins(SbtOsgi)
OsgiKeys.exportPackage := Seq("sangria.*;version=${Bundle-Version}")
OsgiKeys.privatePackage := Seq()
```